src/integrationtest/python/doc_traversals_tests.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from janusgraph_container import JanusGraphContainer
+from janusgraph_python.driver.janusgraph_remote_connection_builder import JanusGraphRemoteConnectionBuilder
+from gremlin_python.structure.graph import Graph
+
+
+class TestDocTraversals(unittest.TestCase):
+    def setUp(self):
+        self.container = JanusGraphContainer()
+
+        self.client = JanusGraphRemoteConnectionBuilder().connect(host=self.container.get_host_ip(), port=8182,
+                                traversal_source="gods_traversal").get_connection()
+
+        self.container.start()
+        self.g = Graph().traversal().withRemote(self.client)
+
+    def tearDown(self) -> None:
```

Comment:
This is using type hint annotations which were added only in Python 3.5: https://docs.python.org/3/library/typing.html

Do we really need this annotation? It will prevent working on any Python version prior to 3.5

Reply:
Well @mbrukman : We are supporting currently only `Python > 3.4`. 
Are you sure the `hint annotations` aren't supported prior to 3.5 because the build works fine on 3.4 in Travis.
If you check the supported versions in DOCs also, its 3.4,3.5,3.6,3.7 and 3.8. Python 2.x will be depreciated sooner than later and unless there is crunch need for us to support python 2.x we should not support such an older version I my opinion